simple-social-icons.php

```diff
@@ -425,55 +433,60 @@ function update( $newinstance, $oldinstance ) {
 	 *
 	 * Outputs the actual widget on the front-end based on the widget options the user selected.
 	 *
+	 * @param array $args The display args.
+	 * @param array $instance The instance settings.
 	 */
-	function widget( $args, $instance ) {
-
-		extract( $args );
+	public function widget( $args, $instance ) {
 
 		/** Merge with defaults */
 		$instance = wp_parse_args( (array) $instance, $this->defaults );
 
-		echo $before_widget;
+		echo $args['before_widget']; // phpcs:ignore
 
-			if ( ! empty( $instance['title'] ) )
-				echo $before_title . apply_filters( 'widget_title', $instance['title'], $instance, $this->id_base ) . $after_title;
+		if ( ! empty( $instance['title'] ) ) {
+			echo $args['before_title'] . apply_filters( 'widget_title', $instance['title'], $instance, $this->id_base ) . $args['after_title']; // phpcs:ignore
+		}
 
 			$output = '';
 
 			$profiles = (array) $this->profiles;
 
-			foreach ( $profiles as $profile => $data ) {
-
-				if ( empty( $instance[ $profile ] ) )
-					continue;
-
-				$new_window = $instance['new_window'] ? 'target="_blank" rel="noopener noreferrer"' : '';
+		foreach ( $profiles as $profile => $data ) {
 
-				if ( is_email( $instance[ $profile ] ) || false !== strpos( $instance[ $profile ], 'mailto:' ) )
-					$new_window = '';
+			if ( empty( $instance[ $profile ] ) ) {
+				continue;
+			}
 
-				if ( is_email( $instance[ $profile ] ) ) {
-					$output .= sprintf( $data['pattern'], 'mailto:' . esc_attr( antispambot( $instance[ $profile ] ) ), $new_window );
-				} elseif ( 'phone' === $profile ) {
-					$output .= sprintf( $data['pattern'], 'tel:' . esc_attr( antispambot( $instance[ $profile ] ) ), $new_window );
-				} else {
-					$output .= sprintf( $data['pattern'], esc_url( $instance[ $profile ] ), $new_window );
-				}
+			$new_window = $instance['new_window'] ? 'target="_blank" rel="noopener noreferrer"' : '';
 
+			if ( is_email( $instance[ $profile ] ) || false !== strpos( $instance[ $profile ], 'mailto:' ) ) {
+				$new_window = '';
 			}
 
-			if ( $output ) {
-				$output = str_replace( '{WIDGET_INSTANCE_ID}', $this->number, $output );
-				printf( '<ul class="%s">%s</ul>', $instance['alignment'], $output );
+			if ( is_email( $instance[ $profile ] ) ) {
+				$output .= sprintf( $data['pattern'], 'mailto:' . esc_attr( antispambot( $instance[ $profile ] ) ), $new_window );
+			} elseif ( 'phone' === $profile ) {
+				$output .= sprintf( $data['pattern'], 'tel:' . esc_attr( antispambot( $instance[ $profile ] ) ), $new_window );
+			} else {
+				$output .= sprintf( $data['pattern'], esc_url( $instance[ $profile ] ), $new_window );
 			}
+		}
+
+		if ( $output ) {
+			$output = str_replace( '{WIDGET_INSTANCE_ID}', $this->number, $output );
+			printf( '<ul class="%s">%s</ul>', esc_attr( $instance['alignment'] ), esc_html( $output ) );
+		}
 
-		echo $after_widget;
+		echo $args['after_widget']; // phpcs:ignore
```

Comment:
This [isn't escaped](https://github.com/WordPress/wordpress-develop/search?q=%24args%5B%27after_widget%27%5D) in the Core widgets